spec/Mapper/DonorMapperSpec.php

```diff
@@ -93,4 +93,20 @@ function it_can_delete_donor($collection, $donorSchema, Donor $donor, Expression
 
         $this->delete($donor);
     }
+
+    function it_can_find_report_existing_donor(
+        $collection
+    ) {
+        $collection->has('foobar')->willReturn(true);
+
+        $this->shouldHaveKey('foobar');
```

Comment:
That throws a type error. It is expecting byrokrat\giroapp\Mapper\boolean instead of boolean.